eks-distro-base/check_update.sh

```diff
@@ -18,12 +18,14 @@ set -e
 set -o pipefail
 set -x
 
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
```

Comment:
I'm wondering if there are cases where `BASEDIR=$(dirname "$0")` doesn't work

Reply:
This should be fine, this is how we reference it in all of our scripts so if there are cases that this doesn't work, we will have to change all of those anyways